src/site/content/en/learn/accessibility/why/why.assess.yml

```diff
@@ -26,11 +26,11 @@ questions:
   - type: multiple-choice
     cardinality: "1+"
     correctAnswers: "1,2"
-    stem: How might someone try to enact change on the web?
+    stem: What is an effective way to enact change on the web?
     options:
       - content: "Complain on Twitter."
-        rationale: "Complaining may make you feel better, but you can't expect change."
+        rationale: "If you have a large following or you say the right thing at the right time, your tweet may pick up conversation and possibly lead to change. However, for many, Twitter is a way to talk about a problem but not make lasting changes."
       - content: "File a lawsuit."
-        rationale: "This is an option available in many countries, but not all places."
+        rationale: "This is an option available in countries which have accessibility laws. Filing a law suit will bring the attention of a company's lawyers, who may decide it's worth the company's time to address the problem."
```

Comment:
super nitpick: On line 33 it says "lawsuit" without a space, and here on line 34 it says "law suit" with a space. I'd go with "lawsuit", no spaces.